src/backend/src/services/change-requests.services.ts

```diff
@@ -324,6 +326,18 @@ export default class ChangeRequestsService {
       }
     }
 
+    // send a reply to a CR's notifications of its updated status
+    try {
+      const relevantThreads = await prisma.message_Info.findMany({ where: { changeRequestId: foundCR.crId } });
```

Comment:
this doesn't need to be in this try catch, you should pull it out. Also instead of doing it this way, you should do an if statement and only try to send the slack messages inside the if statement if relevant threads isn't undefined or an empty array. I know what you do here technically works, but this way is easier to read